roles/edpm_neutron_ovn/defaults/main.yml

```diff
@@ -55,6 +55,7 @@ edpm_neutron_ovn_agent_rootwrap_DEFAULT_rlimit_nofile: '1024'
 # neutron-ovn-agent.conf
 edpm_neutron_ovn_agent_DEFAULT_debug: 'True'
 edpm_neutron_ovn_agent_agent_root_helper: 'sudo neutron-rootwrap /etc/neutron/rootwrap.conf'
+edpm_neutron_ovn_agent_agent_extensions: 'metadata'
```

Comment:
but setting this as default should be issue for antelope(current default) as that extension was not backported there, no?